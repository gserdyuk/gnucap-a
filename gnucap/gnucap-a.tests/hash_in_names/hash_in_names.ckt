* hash in names

.param RSH=50k
.param R# = 50K

r1   1  0  50k
r#   1  0  50K

r##  1  #1 50k
r3#  #1 0  RSH

R4   1  2  R#
R5   2  0  50k

v1   1  0  dc 2

.list
.print op v(*)
.op 
.end

