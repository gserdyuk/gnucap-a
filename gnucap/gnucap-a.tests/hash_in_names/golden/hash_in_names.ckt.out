Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-04-09
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* hash in names 
r1 ( 1 0 )  50.K
r# ( 1 0 )  50.K
r## ( 1 #1 )  50.K
r3# ( #1 0 ) RSH
R4 ( 1 2 ) R#
R5 ( 2 0 )  50.K
v1 ( 1 0 )  DC  2.
#           v(#1)      v(1)       v(2)       v(r1)      v(r#)      v(r##)     v(r3#)     v(R4)      v(R5)      v(v1)     
 27.        1.         2.         1.         2.         2.         1.         1.         1.         1.         2.        
